packages/web-client/package.json

```diff
@@ -161,6 +161,7 @@
     "extends": "../../package.json"
   },
   "dependencies": {
+    "@cardstack/ember-helpers": "^0.0.1",
```

Comment:
It used to matter with Ember CLI whether things go in `dependencies` or `devDependencies`, in `safe-tools-client` it’s different from this. But maybe it doesn’t matter anymore?

Should this use the same lockstep version as the rest of the packages?

Reply:
>It used to matter with Ember CLI whether things go in dependencies or devDependencies, in safe-tools-client it’s different from this. But maybe it doesn’t matter anymore?

For apps, I don't think it matters. For addons, it definitely matters.

>Should this use the same lockstep version as the rest of the packages?

Ah, good point.